backend/db/migrate/20230608080932_rename_api_entreprise_scopes_in_enrollments.rb

```diff
@@ -0,0 +1,127 @@
+        when "attestations_fiscales"
+          "attestation_fiscale_dgfip"
+        when "attestations_sociales"
+          "attestation_sociale_urssaf"
+        when "bilans_entreprise_bdf"
+          "bilans_bdf"
+        when "fntp_carte_pro"
+          "open_data"
+        when "certificat_cnetp"
+          "certification_cnetp"
+        when "msa_cotisations"
+          "cotisations_msa"
+        when "certificat_opqibi"
+          "certification_opqibi"
+        when "probtp"
+          "cotisations_probtp"
+        when "qualibat"
+          "open_data"
+        when "certificat_rge_ademe"
+          "open_data"
+        when "exercices"
+          "chiffre_affaires_dgfip"
+        when "extrait_court_inpi"
+          "open_data"
+        when "extraits_rcs"
+          "open_data"
+        when "entreprises"
+          ["mandataires_sociaux_infogreffe", "unites_legales_etablissements_insee"]
+        when "etablissements"
+          ["open_data", "unites_legales_etablissements_insee"]
+        when "liasse_fiscale"
+          "liasses_fiscales_dgfip"
+        when "actes_inpi"
+          "open_data"
+        when "bilans_inpi"
+          "comptes_annuels_inpi"
+        when "conventions_collectives"
+          "open_data"
+        when "effectifs_acoss"
+          "effectifs_urssaf"
+        when "entreprises_artisanales"
+          "open_data"
+        when "eori_douanes"
+          "open_data"
+        when "certificat_agence_bio"
+          "open_data"
+        else
+          scope
+        end
+      end
+
+      enrollment.update!(scopes: scopes.flatten)
+    end
+  end
+
+  def down
+    valid_enrollments.find_each do |enrollment|
+      scopes = enrollment.scopes.map do |scope|
+        case scope
+        when "unites_legales_etablissements_insee"
+          ["entreprises", "etablissements"]
+        when "associations_djepva"
+          "associations_donnees_protegees"
+        when "mandataires_sociaux_infogreffe"
+          "entreprises"
+        when "chiffre_affaires_dgfip"
+          "exercices"
+        when "comptes_annuels_inpi"
+          "bilans_inpi"
+        when "bilans_bdf"
+          "bilans_entreprise_bdf"
+        when "liasses_fiscales_dgfip"
+          "liasse_fiscale"
+        when "attestation_fiscale_dgfip"
+          "attestations_fiscales"
+        when "attestation_sociale_urssaf"
+          "attestations_sociales"
+        when "cotisations_msa"
+          "msa_cotisations"
+        when "cotisations_probtp"
+          "probtp"
+        when "certification_cnetp"
+          "certificat_cnetp"
+        when "open_data"
+          [
+            "associations",
+            "fntp_carte_pro",
+            "qualibat",
+            "certificat_rge_ademe",
+            "extrait_court_inpi",
+            "extraits_rcs",
+            "actes_inpi",
+            "conventions_collectives",
+            "entreprises_artisanales",
+            "eori_douanes",
+            "certificat_agence_bio"
```

Comment:
Certification OPQIBI est open data, ajouter dans la liste

```suggestion
            "certificat_agence_bio",
            "certificat_opqibi"
```